test/zzz-correct-index-names.sh

```diff
@@ -47,7 +47,12 @@ for project in $OPS_NAMESPACES ; do
         -p TEST_POD_SLEEP_TIME=1 \
         -p TEST_NAMESPACE_NAME=${project} \
         -p TEST_ITERATIONS=1 | oc create -f - 2>&1 | artifact_out
-    os::cmd::try_until_text "oc get -n ${project} pods test-pod" "^test-pod.* Running " "${timeout}"
+    if os::cmd::try_until_text "oc get -n ${project} pods test-pod" "^test-pod.* Running " "${timeout}"; then
+       os::log::debug "test-pod successfully deployed to ${project}"
+    else
+       oc describe -n ${project} test-pod 2>&1 | artifact_out
+       oc describe node 2>&1 | artifact_out
```

Comment:
need to return 1 or exit 1 from here because the rest of the test will either fail or return strange unexpected results